test/functional/collection_tests.js

```diff
@@ -12,1648 +11,1040 @@ describe('Collection', function() {
     return setupDatabase(this.configuration);
   });
 
-  /**
-   * @ignore
-   */
-  it('should correctly execute basic collection methods', {
-    metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
-    },
-
-    // The actual test we wish to run
-    test: function(done) {
-      var self = this;
-      var client = self.configuration.newClient(self.configuration.writeConcernMax(), {
+  describe('tests with standard configuration', function() {
```

Comment:
It would be great if we can come up with a better name here. Perhaps `standard collection tests` or something like that?